src/libbluechi/socket.c

```diff
@@ -5,13 +5,12 @@
 #include <sys/socket.h>
 #include <unistd.h>
 
-#include "libbluechi/common/common.h"
 #include "socket.h"
 
 int create_tcp_socket(uint16_t port) {
         struct sockaddr_in6 servaddr;
 
-        _cleanup_fd_ int fd = socket(AF_INET6, SOCK_STREAM | SOCK_CLOEXEC | SOCK_NONBLOCK, 0);
+        int fd = socket(AF_INET6, SOCK_STREAM | SOCK_CLOEXEC | SOCK_NONBLOCK, 0);
```

Comment:
Please revert this change, having `_cleanup_fd_` here is intentional: https://github.com/containers/bluechi/pull/602